routes/contact.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+
+const router = express.Router()
+
+const Contact = require('../models/contact')
+
+router.get('/', async (req, res) => {
+    res.render('../views/contactus')
+})
+
+router.post('/', async (req, res) => {
+    const contact = new Contact ({
+        name : req.body.name,
+        email : req.body.email,
+        subject : req.body.subject,
+        text : req.body.text
+    })
+    try{
+    res.status(201).json(contact)
```

Comment:
looks like there is no insertion of record in this API

Reply:
Added contact.save() for insertion